<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>network_setting</class>
 <widget class="QFrame" name="network_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <property name="lineWidth">
   <number>0</number>
  </property>
  <widget class="QLabel" name="label_ip_address">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>IP Address</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_gw_ip">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gateway IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_netmask_ip">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NetMask IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledit_ip_address">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledit_gw_ip">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>150</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledit_netmask_ip">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>250</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Save Settings</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="controller.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="controller.qrc"/>
 </resources>
 <connections/>
</ui>
